src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import InitialsPropType from '../../prop-types/initials';
+import ListPropType from '../../prop-types/list';
+import './Avatar.scss';
+
+const propTypes = {
+    name: PropTypes.string.isRequired,
+    initials: InitialsPropType,
+    src: PropTypes.string,
+    size: ListPropType(['xs', 'sm', 'md', 'lg']),
+    theme: ListPropType(['light', 'dark']),
+    status: PropTypes.string,
+    hasNotification: PropTypes.bool,
+    hasHalo: PropTypes.bool,
+    onClick: PropTypes.func,
+    tabIndex: PropTypes.number,
+    className: PropTypes.string,
+    style: PropTypes.objectOf(PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.number,
+    ])),
+};
+
+const defaultProps = {
+    initials: null,
+    src: null,
+    size: 'md',
+    theme: 'light',
+    status: null,
+    hasNotification: false,
+    hasHalo: false,
+    onClick: null,
+    tabIndex: 0,
+    className: null,
+    style: null,
+};
+
+class Avatar extends Component {
+    avatarRef = null
+
+    handleClick = (event) => {
+        const propsOnClick = this.props.onClick;
+        if (propsOnClick) {
+            propsOnClick(event);
+            this.avatarRef.blur();
+        }
+    }
+
+    handleKeyDown = (event) => {
+        if (event.key === 'Enter' || event.key === ' ') {
+            event.preventDefault();
+            this.handleClick(event);
+        }
+    }
+
+    // NB: try not to rely on this fn
+    //  in the future it may be removed
+    calculateInitials = name => (name ? name.match(/\b\w/g).slice(0, 3).join('') : '?')
+
+    render() {
+        const imageOrInitials = this.props.src
+            ? <img src={this.props.src} alt={`${this.props.name} profile`} />
+            : this.props.initials || this.calculateInitials(this.props.name);
+        const notification = this.props.hasNotification
+            ? (
+                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                    <g fill="none" fillRule="evenodd" transform="translate(3 3)">
+                        <circle cx="9" cy="9" r="8" stroke="#F33061" strokeWidth="2" />
+                        <path fill="#F33061" d="M8 5h2v5H8zM8 11h2v2H8z" />
+                    </g>
+                </svg>
```

Comment:
does it have sense to have a separate component for this SVG, will be reused?
See Button branch, and IconArrow component

Reply:
@jfrej and I have flagged icons as a point to discuss... I'll leave this SVG here for now as I believe the only place the notification icon will be used is in the Avatar... 

If we have a need for the SVG elsewhere then we could make it part of the Icon component

Reply:
ok